docs/tutorial/using-docker-container.md

```diff
@@ -99,21 +88,13 @@ apt-get update
 
 with this the local Windows and WSL are all installed. To check `usbipd` tool is working well on both side, please follow the following steps:
 
-1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd wsl list`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_wsl_l.png" alt="" height="">
-
-   as you can see, all USB devices from Windows have been found and not attached sate.
-
-2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind -b <BUSID>`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_bind.png" alt="" height="">
+1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd list` for a list of USB serial devices.
 
-   **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id I would like to bind.
+2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind --busid <BUSID>`:
 
-3. after binding, please attach the specified device to WSL with `usbipd wsl attach --busid 1-1` command in the powershell command prompt.
+   **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id `<BUSID>` I would like to bind.
 
-<img src="../../media\tutorials\using_docker_container\usbipd_wsl_attach.png" alt="" height="">
+3. after binding, please attach the specified device to WSL with `usbipd attach --wsl --busid <BUSID>` command in the powershell command prompt.
```

Comment:
We should keep consistency here, so we should keep using "PowerShell" instead of "powershell"